Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -33,17 +33,12 @@
 NSString * const BRANCH_PREFS_KEY_SESSION_PARAMS = @"bnc_session_params";
 NSString * const BRANCH_PREFS_KEY_INSTALL_PARAMS = @"bnc_install_params";
 NSString * const BRANCH_PREFS_KEY_USER_URL = @"bnc_user_url";
```

Comment:
ah, right. I was searching for `bnc_user_url` instead of `BRANCH_PREFS_KEY_USER_URL` :facepalm